tasks.py

```diff
@@ -6,6 +6,7 @@
 import sys
 import yaml
 import tempfile
+
```

Comment:
Can you please re-do this commit without all of the unrelated formatting changes?  @cgoncalves